src/app/shared/palette/label/label-field.html

```diff
@@ -1 +1 @@
-<dl class="case-field"><dt><ccd-markdown [content]="caseField.label"></ccd-markdown></dt><dd></dd></dl>
+<dl class="case-field"><dt><ccd-markdown [content]="caseField.value || caseField.label"></ccd-markdown></dt><dd></dd></dl>
```

Comment:
aren't we breaking encapsulation here? it's the caseField itself that should know how to return it's label. Can we add a method in caseField:
`getLabel() {
  return caseField.value || caseField.label
}



label-substitutor.directive.ts

Reply:
Same as above, this is an exception of the `LabelField`; this is not something that should be done for any other type of fields. As such, this logic cannot belong to `CaseField`.